_topic_map.yml

```diff
@@ -2342,6 +2342,8 @@ Topics:
       File: virt-uploading-local-disk-images-virtctl
     - Name: Uploading a local disk image to a block storage DataVolume
       File: virt-uploading-local-disk-images-block
+    - Name: Managing Offline virtual machine snapshots
```

Comment:
Looks like "offline" is lowercase elsewhere in the PR, assume this should be lower too here

Reply:
Good catch! Fixed it.